tests/sub_group_mask/sub_group_mask_all.cpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Extension Conformance Test
```

Comment:
I'm a bit confused as well, this is an Intel extension, not a Khronos one, why would it be in the CTS? In any case the CTS would probably need some extra mechanism to enable/disable SYCL extensions, here's it's just added in full.

Reply:
According to my understanding this is not "an Intel extension", but rather `oneAPI` extension, which is a cross-vendor extension targeting future core specification versions. What would be the right process to add tests for such extensions?

Reply:
Let's discuss this during the next SYCL committee.